packages/manager/modules/vps/src/dashboard/tile/configuration/upgrade/routing.js

```diff
@@ -239,7 +238,17 @@ export default /* @ngInject */ ($stateProvider) => {
               },
             );
           })
+          .then(() => {
+            atInternet.trackPage({
+              name: `vps::upgrade-confirm-banner::success::${upgradeType}-${from}-to-${to}`,
+              type: 'navigation',
+            });
+          })
           .catch((error) => {
+            atInternet.trackPage({
+              name: `vps::upgrade-confirm-banner::error::${upgradeType}-${from}-to-${to}`,
+              type: 'navigation',
+            });
```

Comment:
In packages/manager/modules/vps/src/dashboard/vps-dashboard.routing.js you have a function trackPage why you don't reuse it here ?